python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -791,6 +806,9 @@ async def configure(self, config):
 
         self.take_detection_image = config.take_detection_image
 
+        # Assume the time-on-target is 10 minutes
+        self.time_on_target = 10 * 60
```

Comment:
can you do `self.time_on_target = 600.0` instead?  

Also, doesn't seem like this Is a configuration parameter. Maybe move the definition to the class `__init__` method?